### PR TITLE
skeleton html

### LINKED ISSUE #3: Testing github Branch/Pull/Merge
**Github commands for creating new branches, checking into them and merging.** 
- _To create a new branch type the command: git branch [write the name of branch here] i.e branch name = HTML_

- _To push files added to branch, example creating index.html. Use the command: git push origin [HTML]_
 
- _To take you to that branch type the command: git checkout [name of branch]_

- _To add a new branch and go to it in one command, type: git checkout -b [write name of branch here]_

- _To merge to a branch, go into the branch you'll be merging using git checkout. Then type the command: git merge [name of branch you want to merge into] i.e Branch HTML to main. git checkout HTML; git merge main_

- _You can also merge branches using the command: git rebase [name of branch]
Branch HTML to main. git checkout HTML; git rebase main_

### PR DESCRIPTION
**Pull request by ffakih5**
Added skeleton html. testing pull request

### CODE REVIEW

**Review by ffakih5 — COMMENTED** [review]
approved

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+     <head lang="en">
+     <meta charset="UTF-8">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+          <title> Travelmate</title>
+     </head>
+     <body>
+     <h1>
```
Comment (malakhim) [inline index.html]: Don't forget to close your `h1`!

```suggestion
     <h1></h1>
```

**Review by malakhim — CHANGES_REQUESTED** [review]

**Comment by ffakih5** [comment]
`

**Comment by ffakih5** [comment]
Need to merge

**Comment by ffakih5** [comment]
changed to draft

**Pull request closed without merging**